components/brave_new_tab_ui/api/background.ts

```diff
@@ -18,6 +19,19 @@ export const randomBackgroundImage = (): NewTab.BackgroundWallpaper => {
   return image
 }
 
+export const randomColorBackground = (color: string): NewTab.BackgroundWallpaper => {
+  console.assert(color === BraveNewTabPage.RANDOM_SOLID_COLOR_VALUE || color === BraveNewTabPage.RANDOM_GRADIENT_COLOR_VALUE)
+
+  let targetColors = color === BraveNewTabPage.RANDOM_SOLID_COLOR_VALUE ? solidColorsForBackground : gradientColorsForBackground
+  const randomIndex: number = Math.floor(Math.random() * targetColors.length)
+  const randomColor: NewTab.BackgroundWallpaper = {
```

Comment:
I think it's safe to leave this type of - because the function already specifies the return type typescript will complain if it doesn't line up :smile